client/src/main/java/gui/printers/FieldPrinter.java

```diff
@@ -23,40 +31,83 @@ public static void insertFields(GridPane printingBoard) {
         addToEachSpace(numberOfRows, numberOfColumns, printingBoard);
     }
 
-    private static Button createField() {
+    private static void addToEachSpace(int numberOfRows, int numberOfColumns, GridPane printingBoard) {
+        for (int widthPosition = 0; widthPosition < numberOfRows; widthPosition++) {
+            for (int heightPosition = 0; heightPosition < numberOfColumns; heightPosition++) {
```

Comment:
IntStreams would be great here!

Reply:
Thanks it's great idea for future refactor. :+1: